File: Python/repetitivas/fatorial.py
# Exercício: Fatorial
# Objetivo: calcular o fatorial de um número inteiro.

fatorial = 1

n = int(input("Digite o valor de N: "))

for i in range(1, n+1):
    fatorial = fatorial * i

print(f"FATORIAL = {fatorial}")

File: Java/Estruturas Repetitivas/fatorial.java
// Exercício: Fatorial
// Objetivo: calcular o fatorial de um número inteiro informado pelo usuário.

import java.util.Scanner;

public class fatorial {
    public static void main(String[] args) {
        
        Scanner sc = new Scanner(System.in);

        int n, i, fat;

        System.out.print("Digite o valor de N: ");
        n = sc.nextInt();

        fat = 1;
        for (i = 1; i <= n; i++) {
            fat = fat * i;
        }

        System.out.println("FATORIAL = " + fat);

        sc.close();
    }
}